tokens.xml

```diff
@@ -695,6 +695,20 @@ Creature tokens you control have flying and vigilance.</text>
             <token>1</token>
             <tablerow>2</tablerow>
         </card>
+        <card>
+            <name>Beeble Token</name>
+            <prop>
+                <colors>U</colors>
+                <type>Token Creature — Beeble</type>
+                <maintype>Creature</maintype>
+                <pt>1/1</pt>
+            </prop>
+            <set picURL="https://c1.scryfall.com/file/scryfall-cards/large/front/9/b/9bbfa4d4-106b-4fec-86ac-b81b71a8f432.jpg?1584741596">UND</set>
+            <reverse-related>B.O.B. (Bevy of Beebles)</reverse-related>
```

Comment:
```suggestion
```
bob makes 4 beebles, it doesn't have an option to make only one

Reply:
The option to make only one is to satisfy the unique way B.O.B. handles loyalty counters:
"The number of loyalty counters on B.O.B. is equal to the number of Beebles you control. (Create or sacrifice Beebles whenever B.O.B. gains or loses loyalty.)"
Since you can add one loyalty with it's first loyalty ability, you need to be able to create one Beeble at a time.

Reply:
that's a good point, I didn't interpret the card fully